### PR TITLE
titan: update the step of disabling titan

### PR DESCRIPTION
**Pull request by TomShawn**
### What is changed, added or deleted? (Required)
titan: update the step of disabling titan

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [ ] master (the latest development version)
- [ ] v6.3 (TiDB 6.3 versions)
- [x] v6.1 (TiDB 6.1 versions)
- [x] v5.4 (TiDB 5.4 versions)
- [x] v5.3 (TiDB 5.3 versions)
- [x] v5.2 (TiDB 5.2 versions)
- [x] v5.1 (TiDB 5.1 versions)
- [x] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: storage-engine/titan-configuration.md [hunk 1]
```diff
@@ -118,7 +118,7 @@ To disable Titan, you can configure the `rocksdb.defaultcf.titan.blob-run-mode`
 - When the option is set to `read-only`, all newly written values are written into RocksDB, regardless of the value size.
 - When the option is set to `fallback`, all newly written values are written into RocksDB, regardless of the value size. Also, all compacted values stored in the Titan blob file are automatically moved back to RocksDB.
 
-To disable Titan, set `blob-run-mode = "fallback"` and perform a full compaction using tikv-ctl. After that, check the monitoring metrics, confirm that the blob file size decreases to `0`. Then you can set `rocksdb.titan.enabled` to `false` and restart TiKV.
+To disable Titan, set `blob-run-mode = "fallback"` and `merge-small-file-threshold="0KB"`, and then perform a full compaction using tikv-ctl. After that, check the monitoring metrics, confirm that the blob file size decreases to `0`. Then you can set `rocksdb.titan.enabled` to `false` and restart TiKV.
```
Comment (tabokie) [inline storage-engine/titan-configuration.md:121]: Also add `discardable-ratio = 1.0`

Comment (TomShawn) [reply storage-engine/titan-configuration.md]: addressed

**Review by tabokie — APPROVED** [review]

File: storage-engine/titan-configuration.md [hunk 2]
```diff
@@ -118,7 +118,31 @@ To disable Titan, you can configure the `rocksdb.defaultcf.titan.blob-run-mode`
...
-To disable Titan, set `blob-run-mode = "fallback"` and perform a full compaction using tikv-ctl. After that, check the monitoring metrics, confirm that the blob file size decreases to `0`. Then you can set `rocksdb.titan.enabled` to `false` and restart TiKV.
+To disable Titan, perform the following steps:
+
+1. Update the configuration file of the TiKV nodes you wish to disable Titan for and restart TiKV:
+
+    ```toml
+    [rocksdb.defaultcf.titan]
+    blob-run-mode = "fallback"
+    discardable-ratio = 1.0
+    merge-small-file-threshold="0KB"
```
Comment (tabokie) [inline storage-engine/titan-configuration.md:129]:
```suggestion
    merge-small-file-threshold = "0KB"
```

**Review by tonyxuqqi — APPROVED** [review]

**Pull request merged**